Classes/Private/Manager/Objects/OCTCall.m

```diff
@@ -29,4 +29,9 @@ - (NSDate *)onHoldDate
     return [NSDate dateWithTimeIntervalSince1970:self.onHoldStartInterval];
 }
 
+- (BOOL)isPaused
+{
+    return ! (self.pausedStatus == OCTCallPausedStatusNone);
```

Comment:
This method doesn't use any private properties of OCTCall. I think that you can move it to separate category file `OCTCall+Private.m`